tests/Feature/QueryBuilder/FilterTest.php

```diff
@@ -17,7 +17,7 @@ public function it_can_filter_on_associations()
         $post = factory(Post::class)->make();
         $users->first()->posts()->save($post);
 
-        $request = $this->request()->fields('id')->filter('posts', [$post->id])->make();
+        $request = $this->request()->fields('id')->filter('posts', $post->id)->make();
```

Comment:
Could you revert this filter to expect an array? I want to ensure that the association filter also works on arrays of input. Furthermore, the name `posts` already implies that it expects an array.

Reply:
When in an array it gives an error 
`Apitizer\Exceptions\InvalidInputException: Expected filters[posts] to receive [string] got [array]`

Reply:
Exactly, so the definition of the filter should also be changed.

Reply:
Filter in the userbuilder adjusted to once again support arrays